### DISCUSSION TITLE
Raxda Orion O6

### DISCUSSION THREAD

**Discussion by derTim** [geerlingguy/sbc-reviews#61] [General]
Hi Jeff,

I just read about the new Raxda Orion O6 and instantly preordered it, as I want to have an ARM based desktop replacement for years, and everytime I tried in the past, I gave up.
Pi4, Khadas VIM, either it was too slow in general, or video playback sucked or I had issues with even getting my monitor (UWQHD back then, DualQHD now) usable.

And now out of nowhere, there is a new SBC that sounds very promising, 12 Core ARM v9, PCIE Slot for a GPU, dual 5Gbit Networking (if it won't work as a desktop replacement, i can add an HBA and use it as a NAS).
More Details: [CNX](https://www.cnx-software.com/2024/12/18/radxa-orion-o6-mini-itx-motherboard-is-powered-by-cix-p1-12-core-armv9-soc-with-a-30-tops-ai-accelerator/)

Any chance you are also getting one of those? 😄

**Comment by geerlingguy** [comment]
Indeed, I ordered one the same hour I found out about it :)

New issue added: https://github.com/geerlingguy/sbc-reviews/issues/62
